bundles/org.openhab.binding.venstarthermostat/README.md

```diff
@@ -0,0 +1,60 @@
+# Venstar Thermostat Binding
+
+The Venstar Thermostat binding supports an interface to WiFi enabled 
+ColorTouch and Explorer thermostats manufactured by Venstar[1].
+
+## Prerequisites
+
+Venstar WiFi enabled thermostats provide a local API that this binding uses
+to communicate with the thermostat. This binding does not require "cloud" 
+access and may be used independently of Venstar's Skyport cloud services.
+
+The Local API is not enabled by default, so you will need to set up your 
+thermostat by configuring its WiFi connection and enabling the Local API. In
+order for the binding to connect, you will need to enable HTTPS support and 
+set a username and password. While it is possible to enable the Local API
+without HTTPS and authentication, the binding doesn't support it, in an effort
+to provide as secure an installation as possible.
+
+When you've set the username and password, make a note of these, as you'll need
+to enter them in the thermostat configuration in OpenHAB.
+
+
+## Usage
+
+### Discovery
+
+Once the binding is installed it will attempt to auto discovery Venstar thermostats located on the local network.
+These will appear as Things in the system Inbox. 
+After adding the Inbox item, enter the user name and password from the physical thermostat in the Thing's configuration.
+
+### Channels
+
+| Channel            | Type               | Description                  | Notes                                                  |
+|-:------------------|-:------------------|-:----------------------------|-:------------------------------------------------------|
```

Comment:
If I show the page on GitHub this table doesn't render correctly. Could it be due to the colons `:` 
```
|-:------------------|-:------------------|-:----------------------------|-:------------------------------------------------------|
```